tasks/cert.yaml

```diff
@@ -31,7 +33,7 @@
 
 - name: Starting paused Services
   service: name="{{item.item}}" state=started
-  when: (item.failed != true) and (item.state == "stopped")
+  when: item.state == "stopped"
```

Comment:
Yeah, makes sense. Probably that service failed because of the missing certificate.